test/e2e/kustomize_test.go

```diff
@@ -116,8 +116,9 @@ func TestSyncStatusOptionIgnore(t *testing.T) {
 func TestKustomizeSSHRemoteBase(t *testing.T) {
 	Given(t).
 		// not the best test, as we should have two remote repos both with the same SSH private key
-		SSHRepo().
-		Path("remote-base").
+		SSHRepoURLAdded().
```

Comment:
`SSHRepo()`?

Reply:
I would prefer to keep `SSHRepoURLAdded`. This method only ensures that SSH URL of test repo  is added to argocd, but does not change repo URL which used for app creation.

We why we are verifying cases when ssh repo is added but we use https url etc.